include/openspace/util/spicemanager.h

```diff
@@ -993,6 +993,55 @@ class SpiceManager {
         AberrationCorrection aberrationCorrection, double ephemerisTime,
         int numberOfTerminatorPoints);
 
+    /**
+     * The structure retuned by the #lightTime method.
+     */
+    struct LightTimeResult {
+        /// Epoch of the signal at the target.
+        /// This is the time at which the signal would arrive at the target.
+        double epochAtTarget = 0.0;
+
+        /// Time between transmit and receipt of the signal.
+        /// This is the light travel time.
+        double elapsed = 0.0;
+    };
+
+    /**
+     * This method returns time of a signal recieved at a target from an \observer
+     * corrected for light travel time.
```

Comment:
```suggestion
     * This method returns time of a signal received at a target from an \p observer
     * corrected for light travel time.
```

Which time is this? The `LightTimeResult` returns back two distinct times